runtime/shared_common/CompositeCache.cpp

```diff
@@ -5538,15 +5538,8 @@ SH_CompositeCacheImpl::clearCacheHeaderFullFlags(J9VMThread *currentThread)
 bool
 SH_CompositeCacheImpl::isAllRuntimeCacheFullFlagsSet(void) const
 {
-	if ((0 != (*_runtimeFlags & J9SHR_RUNTIMEFLAG_BLOCK_SPACE_FULL)) &&
-		(0 != (*_runtimeFlags & J9SHR_RUNTIMEFLAG_AOT_SPACE_FULL)) &&
-		(0 != (*_runtimeFlags & J9SHR_RUNTIMEFLAG_JIT_SPACE_FULL)) &&
-		(0 != (*_runtimeFlags & J9SHR_RUNTIMEFLAG_AVAILABLE_SPACE_FULL))
-	) {
-		return true;
-	} else {
-		return false;
-	}
+	return J9_ARE_ALL_BITS_SET(*_runtimeFlags, J9SHR_RUNTIMEFLAG_BLOCK_SPACE_FULL | J9SHR_RUNTIMEFLAG_AOT_SPACE_FULL
+		| J9SHR_RUNTIMEFLAG_JIT_SPACE_FULL | J9SHR_RUNTIMEFLAG_AVAILABLE_SPACE_FULL);
```

Comment:
This is only equivalent because each of the flag constants is exactly one bit, but why is the change part of this PR?

Reply:
Without this change there was a xlC Internal abort on the 32-bit build, see the internal vmfarm build job_output.php?id=6534893
```
    1500-004: (U) INTERNAL COMPILER ERROR while compiling SH_CompositeCacheImpl::isAllRuntimeCacheFullFlagsSet() const.  Compilation ended.  Contact your Service Representative and provide the following information: Internal abort. For more information visit: http://www.ibm.com/support/docview.wss?uid=swg21110810
1586-346 (U) An error occurred during code generation.  The code generation return code was 1.
```

Reply:
Ouch!